go/test-docker/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM alpine:3.22.1
 RUN echo "Building step 1..."
-RUN echo "Building step 2..."  
+RUN echo "Building step 2..."
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)